libraries/AP_GPS/AP_GPS.h

```diff
@@ -651,6 +704,31 @@ class AP_GPS
     // they should be configuring the GPS to run at
     uint16_t get_rate_ms(uint8_t instance) const;
 
+#if AP_MAVLINK_MSG_GNSS_INTEGRITY_ENABLED
+    // general errors in the GNSS system
+    uint32_t get_system_errors(uint8_t instance) const {
```

Comment:
Not a blocker but one suggestion that was raised on the dev call that that it might be worth combining all of these get functions into one function as they are all called in once in the same currently.